cluster/mesos/docker/mesos-slave/bin/wrapdocker

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+# Copyright 2013-2014 Jérôme Petazzoni, Johan Haleby, lalyos, James Harris,
```

Comment:
I'm sorry, we can't take this in the main repo without CLAs signed by all the authors. You have two options at this point: (a) We can possibly take it in `third_party`, or (b) rewrite.